rust/docker-browser-tests/src/shared.ts

```diff
@@ -0,0 +1,29 @@
+import { parse } from 'ts-command-line-args';
+import puppeteer, { Browser, HTTPResponse } from 'puppeteer';
+
+interface IArgs {
+  debugPort: number;
+  url: string;
+}
+
+export const args = parse<IArgs>({
+  debugPort: Number,
+  url: String
+});
```

Comment:
I think shared constants are somewhat weird, I'd make this a function to ensure no code runs on _import_ time of the script.

Reply:
This was taken from the lib example code, I think this might be common practice on javascript